noir-projects/aztec-nr/aztec/src/test/helpers/test_environment.nr

```diff
@@ -74,26 +70,41 @@ impl TestEnvironment {
         test_account.address
     }
 
-    fn create_account_contract(self, secret: Field) -> AztecAddress {
+    fn create_account_contract(&mut self, secret: Field) -> AztecAddress {
         let public_keys = cheatcodes::derive_keys(secret);
-        let args = &[public_keys.ivpk_m.x, public_keys.ivpk_m.y];
-        let address = cheatcodes::deploy(
+        let args = [public_keys.ivpk_m.x, public_keys.ivpk_m.y];
+        let instance = cheatcodes::deploy(
             "@aztec/noir-contracts.js/SchnorrAccount",
             "constructor",
-            args,
+            args.as_slice(),
             public_keys.hash().to_field()
         );
         cheatcodes::advance_blocks(1);
```

Comment:
No, in some places it makes sense. Naming could be improve though, because we use time traveling to "consolidate" the state and thus being able to read from historical data (for example, we cannot read contract public bytecode until it is deployed and accesible)